src/core/entry/cards/Movie.tsx

```diff
@@ -61,6 +69,9 @@ const MovieCard: FC<Props> = ({ entry, className, children }) => {
           </Typography>
           <LinkDropdown options={options} />
         </div>
+        <Typography css={originaltitle} variant="h5" component="h2" color="textPrimary">
+          {originalName && originalName !== movieName && `${originalName}`}
+        </Typography>
```

Comment:
I would suggest doing something like this so that the h2 tag isn't there when not rendering originalName (indenting is probably bad here but you get the idea)
```suggestion
       {originalName && originalName !== movieName &&
        <Typography css={originaltitle} variant="h5" component="h2" color="textPrimary">
          {originalName}
        </Typography>
        }
```